lib/dor/services/client/marcxml.rb

```diff
@@ -21,6 +23,37 @@ def catkey(barcode:)
 
           raise_exception_based_on_response!(resp)
         end
+
+        # Gets MARCXML corresponding to a barcode or catkey
+        # @param barcode [String] required string representing a barcode
+        # @param catkey [String] required string representing a catkey
+        # @raise [NotFoundResponse] when the response is a 500 with "Record not found in Symphony"
+        # @raise [UnexpectedResponse] on an unsuccessful response from the server
+        # @return [Nokogiri::XML::Document] MARCXML
```

Comment:
Why do we have to return a Nokogiri document?  How about just the string containing xml.  Nokogiri is a heavy dependency and I don't want to require it every  place that uses this gem.

Reply:
It is already a dependency of this gem: https://github.com/sul-dlss/dor-services-client/blob/master/dor-services-client.gemspec#L29

Reply:
On one hand, I'm not sure any of our codebases don't **already** have NG as a dependency. On the other, I support keeping NG out of the interface if we can avoid it.

Reply:
I'll be that. Done.